js/mojs-player.babel.js

```diff
@@ -400,6 +400,12 @@ class MojsPlayer extends Module {
   */
   _onHideStateChange ( isHidden ) {
     this._props.isHidden = isHidden;
+    const { onToggleHide } = this._o;
```

Comment:
Let's define default of `null` for the `onToggleHide` option in `_declareDefaults` method. It will be consistent with other options that way. After that we can just use the `this._props.onToggleHide` property.